_data/state_self_employment.yml

```diff
@@ -246,13 +219,10 @@ DC:
   '2013':
     count: 517
     percent: 0.06
+  '2014':
+    count: 554
+    percent: 0.06
 DE:
-  '2004':
-    count: 142
-    percent: 0.03
-  '2005':
-    count: 0
```

Comment:
`self-employment.tsv` shows values of 0, but I'm not sure if that is how it is supposed to be, or a transform issue.